### DISCUSSION TITLE
No Safari support ➾ No use

### DISCUSSION THREAD

**Discussion by richbee0** [phcode-dev/phoenix#1037] [General]
I think the title is quite obvious. We think Phoenix seems like a nice project, but we would like it to work in Safari, or we will not use it.

But I can also add that Google Chrome is absolutely no option to us. We simply loathe it for soo many reasons. Here is one: https://appleinsider.com/articles/23/04/08/reminder-using-chrome-on-a-mac-defeats-apples-privacy-features

This is also why we love Safari.

**Comment by abose** [comment]
Thanks for raising the issue @richbee0 
Safari support is a top priority and should be available by next month.
phcode only became stable in Chrome on December 2022, and currently, we were working on Firefox support here: https://github.com/phcode-dev/phoenix/issues/969

Safari support is going on parallelly, I created an issue to track Safari support here: https://github.com/phcode-dev/phoenix/issues/1038
